torch/csrc/jit/ivalue.h

```diff
@@ -129,10 +135,12 @@ struct IValue {
     std::swap(retainable, rhs.retainable);
     std::swap(tag, rhs.tag);
   }
-  // Accessors for subtypes are arragned together below
+  // Accessors for subtypes are arranged together below
   // While some of these accessors could be generated through templates,
   // we prefer to write them manually for clarity
 
+  Tag getTag() const noexcept { return tag; }
```

Comment:
Don't expose the IValue tag. 
Also can you drop this comment in here:
```
// Note: IValue tags are intentionally private. In the future we may encode this value different (e.g. using NaN boxing), and this would make it more costly to determine the tag for all types vs just determining if something is a particular type. Instead we want clients to use the `isX` methods when possible. If for perf. reasons you really, absolutely, must have a jump table, then we can revisit this.